string robot_name
float64 x
float64 y 
float64 yaw
---
string result